src/auth/auth.resolver.ts

```diff
@@ -39,4 +51,12 @@ export class AuthResolver {
   ) {
     return await this.authService.refreshLoginToken(user, refreshToken.token);
   }
+
+  @Query(() => IsVaild)
+  @UseGuards(GqlAuthGuard)
+  async tokenIsValid(@CurrentUser() user: User) {
+    if (!user) return { isValid: false };
```

Comment:
To sprawdzenie i tak się wykona, bo w przypadku złego tokenu jest wracany Error Unauthorized, ale to już mogę po stronie frontu sprawdzić